### PR TITLE
feat: add users and spaces to grantToken, allow userId in configuration

### PR DESCRIPTION
**Pull request by are**
feat: add users and spaces to grantToken

Allows to specify users and spaces in grantToken method.

feat: allow userId in configuration

Allows to use userId instead of uuid in configuration.

### CODE REVIEW

File: src/core/endpoints/access_manager/grant_token.js [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
-/* eslint camelcase: 0 */
-
-import { GrantTokenInput, GrantTokenObject, ModulesInject } from '../../flow_interfaces';
 import operationConstants from '../../constants/operations';
 
 export function getOperation() {
   return operationConstants.PNAccessManagerGrantToken;
 }
 
+function hasVspTerms(incomingParams) {
```
Comment (mohitpubnub) [inline src/core/endpoints/access_manager/grant_token.js:7]: I see `VSP` word in real life!!! :)

File: test/integration/endpoints/grant_token.test.js [hunk 2]
```diff
@@ -126,36 +109,85 @@ describe('grant token endpoint', () => {
...
+          .catch((err) => {
+            assert.equal(scope.isDone(), false);
+            assert.equal(err.status.message, 'Missing values for either Resources or Patterns.');
+            done();
+          });
+      });
 
-              },
-              groups: {
+      it('should throw when mixing legacy and VSP terms', async () => {
```
Comment (mohitpubnub) [inline test/integration/endpoints/grant_token.test.js:135]: 👍🏻

**Review by mohitpubnub — APPROVED** [review]

File: src/core/endpoints/access_manager/grant_token.js (hunk 1 above)
Comment (are) [reply src/core/endpoints/access_manager/grant_token.js:7]: I wasn't sure if I should use this to be honest, but couldn't figure out anything else :P

**Comment by are** [comment]
@client-engineering-bot release as v7.2.0

**Review by kazydek — APPROVED** [review]

**Pull request merged**